packages/atomic/src/components/common/no-items/tips.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {createTestI18n} from '@/vitest-utils/testing-helpers/i18n-utils';
+import {i18n} from 'i18next';
+import {html} from 'lit';
+import {describe, beforeAll, it, expect} from 'vitest';
+import {renderSearchTips} from './tips';
+
+describe('#renderSearchTips', () => {
+  let i18n: i18n;
+
+  beforeAll(async () => {
+    i18n = await createTestI18n();
+  });
+
+  const renderComponent = async (overrides = {}) => {
```

Comment:
```suggestion
  const renderComponent = async (overrides: Partial<SearchTipsProps> = {}) => {
```

& export the interface...


or just remove the overrides, we're not using them.